src/locale/dim.json

```
@@ -712,7 +713,8 @@
       "Untracked": "Untracked"
     },
     "TriageTab": "Triage",
-    "Vault": "Vault"
+    "Vault": "Vault",
+    "WeaponLevel": "Weapon Lv."
```

```suggestion
    "WeaponLevel": "Weapon Lv. {{levelNumber}}"
```
maybe we could just use the word level? we aren't fighting for space nearly as much as game client is